langchain/embeddings/vertexai.py

```diff
@@ -0,0 +1,33 @@
+"""Wrapper around Google VertexAI embedding models."""
+from typing import Dict, List, Optional
+
+from pydantic import root_validator
+
+from langchain.embeddings.base import Embeddings
+from langchain.llms.vertexai import _VertexAICommon
+from langchain.utilities.vertexai import raise_vertex_import_error
+
+
+class VertexAIEmbeddings(_VertexAICommon, Embeddings):
+    model_name: Optional[str] = "textembedding-gecko"
+
+    @root_validator()
+    def validate_environment(cls, values: Dict) -> Dict:
+        """Validates that the python package exists in environment."""
+        cls._try_init_vertexai(values)
+        try:
+            from vertexai.preview.language_models import TextEmbeddingModel
+        except ImportError:
+            raise_vertex_import_error()
+        values["client"] = TextEmbeddingModel.from_pretrained(values["model_name"])
+        return values
+
+    def embed_documents(self, texts: List[str]) -> List[List[float]]:
+        """Embed search docs."""
```

Comment:
Why are args and returns sections inconsistently provided in this PR?

Reply:
fixed. Docstring actually were inherited from the base class.